staging/src/k8s.io/apiserver/pkg/server/options/server_run_options.go

```diff
@@ -104,11 +105,23 @@ func (s *ServerRunOptions) Validate() []error {
 	if s.TargetRAMMB < 0 {
 		errors = append(errors, fmt.Errorf("--target-ram-mb can not be negative value"))
 	}
-	if s.MaxRequestsInFlight < 0 {
-		errors = append(errors, fmt.Errorf("--max-requests-inflight can not be negative value"))
-	}
-	if s.MaxMutatingRequestsInFlight < 0 {
-		errors = append(errors, fmt.Errorf("--max-mutating-requests-inflight can not be negative value"))
+
```

Comment:
Should we reject / disable this flag if the feature gate isn't on?

Reply:
added. i don't see a feature gate validation in the other existing option/flag in codebase so i didn't at the start. alternatively, can we reject at `config.ApplyTo()`? tho i don't see a difference there..